commit bc77e08478678ff4c075265933d85af9937ace69
Author: Michael Still <mikal@stillhq.com>
Date:   Mon Sep 14 18:32:13 2026 +1000

    Convert the telemetry services to JSON logging.
    
    Ceilometer, Aodh, Gnocchi and CloudKitty now honour
    openstack_logging_json.
    
    Change-Id: Ib82882b8aef7fc9b2ca44db7e1f6e9eac539789f
    Signed-off-by: Michael Still <mikal@stillhq.com>

diff --git a/ansible/roles/aodh/templates/aodh.conf.j2 b/ansible/roles/aodh/templates/aodh.conf.j2
index c9c180385..62242a430 100644
--- a/ansible/roles/aodh/templates/aodh.conf.j2
+++ b/ansible/roles/aodh/templates/aodh.conf.j2
@@ -6,6 +6,7 @@ log_dir = /var/log/kolla/aodh
 log_file = aodh-api.log
 {% endif %}
 debug = {{ aodh_logging_debug }}
+use_json = {{ openstack_logging_json }}
 evaluation_interval = {{ aodh_evaluation_interval }}
 transport_url = {{ rpc_transport_url }}
 
diff --git a/ansible/roles/ceilometer/templates/ceilometer.conf.j2 b/ansible/roles/ceilometer/templates/ceilometer.conf.j2
index b4359f1b1..8f3983646 100644
--- a/ansible/roles/ceilometer/templates/ceilometer.conf.j2
+++ b/ansible/roles/ceilometer/templates/ceilometer.conf.j2
@@ -1,5 +1,6 @@
 [DEFAULT]
 debug = {{ ceilometer_logging_debug }}
+use_json = {{ openstack_logging_json }}
 
 log_dir = /var/log/kolla/ceilometer
 
diff --git a/ansible/roles/cloudkitty/templates/cloudkitty.conf.j2 b/ansible/roles/cloudkitty/templates/cloudkitty.conf.j2
index 0231d46f3..ae1f1fcad 100644
--- a/ansible/roles/cloudkitty/templates/cloudkitty.conf.j2
+++ b/ansible/roles/cloudkitty/templates/cloudkitty.conf.j2
@@ -1,5 +1,6 @@
 [DEFAULT]
 debug = {{ cloudkitty_logging_debug }}
+use_json = {{ openstack_logging_json }}
 
 log_dir = /var/log/kolla/cloudkitty
 
diff --git a/ansible/roles/gnocchi/templates/gnocchi.conf.j2 b/ansible/roles/gnocchi/templates/gnocchi.conf.j2
index 56f74d761..e3f781e29 100644
--- a/ansible/roles/gnocchi/templates/gnocchi.conf.j2
+++ b/ansible/roles/gnocchi/templates/gnocchi.conf.j2
@@ -1,5 +1,6 @@
 [DEFAULT]
 debug = {{ gnocchi_logging_debug }}
+use_json = {{ openstack_logging_json }}
 
 log_dir = /var/log/kolla/gnocchi
 
